client/src/util/pages.jsx

```diff
@@ -2,7 +2,7 @@ import { Page404 } from '../pages/404/404';
 import { PageAbout } from '../pages/About/About';
 import { PageFAQ } from '../pages/FAQ/FAQ';
 import { PageHome } from '../pages/Home/Home';
-import { MaintenancePage } from '../pages/Maintenance/Maintenance';
```

Comment:
This was fixed in dev earlier today, update to latest to avoid conflicts

Reply:
when I merged with dev, it didn't show up as a conflict 🤔... but I hope it's okay!

Reply:
No problem!